README.md

```diff
@@ -210,6 +210,56 @@ helm install my-release marklogic/marklogic --version=1.0.0-ea1 \
 
 We recommend that you use the `values.yaml` file for configuring your installation.
 
+## Cluster Scaling
+
+### Adding Hosts to a Cluster
+
+The MarkLogic Helm chart creates one MarkLogic "host" per Kubernetes pod in a StatefulSet.
+To add a new MarkLogic host to an existing cluster, simply increase the number of pods in your StatefulSet.
+For example, if we want to change the host count of an existing MarkLogic cluster from 2 to 3, run the following Helm command
+
+```
+helm upgrade release-name [chart-path] --namespace name-space --set replicaCount=3
+```
+
+When created, the MarkLogic host will join the existing cluster once the deployment is completed.
+Status can be tracked using the “**kubectl get pods**”. Note that no forests will not be created on the new host.
```

Comment:
Status can be tracked using the “**kubectl get pods**” command.

Reply:
Updated.